scripts/build/ci-deploy/Dockerfile

```diff
@@ -1,9 +1,9 @@
-FROM debian:testing-20210111-slim
```

Comment:
I would rather this be a separate PR. Could have other implications for our builds in general.

Reply:
The main reason for that was that with `debian:testing-20210111` it can't build and install awscli: some problems building PyYAML.
Seems it's related to https://github.com/pypa/pip/issues/11070

I did some more investigation and found that having `pip3 install -U pip==20.0.2` could fix that.

I will update that PR to have more conservative changes.

Reply:
thanks!